_source/reference/attributes.md

```diff
@@ -19,6 +19,7 @@ The following data attributes can be applied to elements to customize Turbo's be
 * `data-turbo-cache="false"` removes the element before the document is cached, preventing it from reappearing on restoration Visits.
 * `data-turbo-eval="false"` prevents inline `script` elements from being re-evaluated on Visits.
 * `data-turbo-method` changes the link request type from the default `GET`. Ideally, non-`GET` requests should be triggered with forms, but `data-turbo-method` might be useful where a form is not possible.
+* `data-turbo-stream="true"` specifies that a link or form should request a Turbo Streams response. Turbo [automatically requests stream responses](/handbook/streams#streaming-from-http-responses) for form submissions with non-`GET` methods; `data-turbo-stream` allows Turbo Streams to be used with `GET` requests as well.
```

Comment:
@ghiculescu thanks, that's a good point. I think your wording of "can accept" makes this clearer. I've updated the description to use that.

I've also added a note in the Handbook section that it references to explain in a bit more detail, which also should make this option easier to discover.

What do you think?

Reply:
@jusko there should still only be one request. Adding `data-turbo-stream` means that request can accept either a stream response, or a regular HTML response.

Reply:
:thinking: Forgive my ignorance @kevinmcconnell... I only glanced at the turbo code quickly, but using it in my client code produces the following:

![image](https://user-images.githubusercontent.com/812279/180514465-c31b018f-16bc-4776-830c-013c80fc8e2a.png)

The last two requests are:
2. the turbo stream response after clicking a link with `data-turbo-stream` (working as expected)
3. A 404 response to a separate request sent with `Content-Type: text/html` in the headers (also sent when clicking the link).

I understand this isn't really the place to discuss this, but on the off chance anyone else experiences this, it's worth noting here. I will look into it a bit more when there's time and update.

Thanks again for the new functionality. Extremely useful!

Reply:
@jusko oh interesting! That definitely sounds like a bug. I hadn’t seen that myself but I will also try to look into it when I get a chance. The intended behaviour is that a single request is sent. Thanks for bringing this up!

I wonder if this is related to the issue that’s being addressed by https://github.com/hotwired/turbo/pull/645.

Also worth mentioning, we’ve discussed changing how `data-turbo-stream` works under the hood (see https://github.com/hotwired/turbo/pull/641#issuecomment-1190328737), and I believe @seanpdoyle is planning to open a PR for that change. So once that’s ready it could be worth testing against that version as well.

Reply:
This might be related to https://github.com/hotwired/turbo/pull/645.